dora/core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -1172,17 +1173,17 @@ public String methodName() {
 
       @Override
       public String toString() {
-        return String.format("path=%s, name=%s, value=%s", path, name, value);
+        return String.format("path=%s, name=%s, value=%s", path, name, Arrays.toString(value));
```

Comment:
nit: we have helpers you can use, it's fine as-is here but you have more options next time
```
return MoreObjects.toStringHelper(this).add(xxx,xxx).toString();
```